mongodb_migrations/base.py

```diff
@@ -5,16 +5,21 @@ class BaseMigration(object):
     def __init__(self,
                  host='127.0.0.1',
                  port='27017',
-                 database=None,
+                 database=None,                 
```

Comment:
we should have this one as a plus, there are extra space in the end

Reply:
sure :)